packages/readabilityjs/test/generate-testcase.js

```diff
@@ -173,7 +173,7 @@ async function fetchSource(url, callbackFn) {
 }
 
 function sanitizeSource(html, callbackFn) {
-  htmltidy(parseHTML(html).serialize(), {
+  htmltidy(html, {
```

Comment:
@sywhb wasn't sure if this is right. It seems like .serialize isn't available here, but i also wasn't sure if parseHTML really needed to be called.

Reply:
You are right! `html` is the HTML string but not a DOM object. @jacksonh